charts/tidb-lightning/templates/job.yaml

```
@@ -37,7 +40,7 @@ spec:
           mountPath: /etc/rclone
         - name: data
           mountPath: /data
-      {{- if and (ne .Values.timezone "UTC") (ne .Values.timezone "") }}
+      {{ if and (ne .Values.timezone "UTC") (ne .Values.timezone "") }}
```

why no dash?

---

I was seeing the following:
```volumeMounts:
        - name: credentials
          mountPath: /etc/rclone
        - name: data
          mountPath: /datarestartPolicy: Never      # if lightning fails, manual intervention is required so no restart```

---

Does the latest tidb-lightning exit with 0 when loading successfully now?

---

@tennix that is what I saw, yes.

---

Then I think we can remove the fail-fast hack script.

---

@tennix I removed the script as well as the fail fast option